README.markdown

```diff
@@ -895,60 +903,16 @@ let playerMark = (player == current ? "X" : "O")
 
 ## Organization and Bundle Identifier
 
-Where an Xcode project is involved, the organization should be set to `Ray Wenderlich` and the Bundle Identifier set to `com.razeware.TutorialName` where `TutorialName` is the name of the tutorial project.
+Where an Xcode project is involved, the organization should be set to `Mindera` or to the client name if required.
 
-![Xcode Project settings](screens/project_settings.png)
+The Bundle Identifier should be set to `com.mindera.client.projectName` where `client` is the client company name and `project` is the name of the project we are working on.
```

Comment:
Perhaps we should explain the reason for this, e.g. it allows the certificates to be automatically created on Mindera's Apple Developer account while avoiding collisions with other projects identifiers.